### DISCUSSION TITLE
Multiline output appears as one long line

### DISCUSSION THREAD

**Discussion by oluckyman** [microsoft/vscode-jupyter#14097] [Questions and Answers]
This code expected to print array with each item on a new line:
```
import os
dir(os)
```
but it dumps it all in one line:

<img width="931" alt="image" src="https://github.com/microsoft/vscode-jupyter/assets/642673/b03223bc-bf37-451d-be99-88734af21227">

here is the expected output as it appears in Jupyter notebook:
<img width="1202" alt="image" src="https://github.com/microsoft/vscode-jupyter/assets/642673/0e0dfd96-270d-4688-bc0e-df8326d70565">

I found Notebook > Output: Word Wrap setting, that helps a little, making it a wall of text instead of multiple lines. But still it renders unusable rapid prototyping in python in VS Code notebooks.

Am I missing something obvious?

**Comment by DonJayamanne** [comment]
@amunger /cc

**Comment by amunger** [comment]
I'm seeing it per line - Can you click the output menu and select 'change presentation' to see if you're using a different renderer? 
this is what I get with the built-in for text/plain
<img width="240" alt="image" src="https://github.com/microsoft/vscode-jupyter/assets/2019016/b0c6bd7f-7636-4aab-bf63-85b69a575166">

**Reply by oluckyman** [reply]
There is only one renderer
<img width="602" alt="image" src="https://github.com/microsoft/vscode-jupyter/assets/642673/652622cc-9da8-43cb-a2eb-e1155b91ded3">

but the strange thing is that today it works as expected!
<img width="949" alt="image" src="https://github.com/microsoft/vscode-jupyter/assets/642673/eab2f13f-6b81-4338-b06c-d78310ee8ebf">


I didn't change anything since yesterday 🤷‍♂️
Happy, it fixed itself 😃
Thank you for the quick response, though!

**Reply by amunger** [reply]
that is strange, perhaps the extension updated. Thanks for the update thoough, glad it's working for you.

**Reply by momodesuka** [reply]
add a line to setting.json
{
"notebook.output.wordWrap": true
}
it works on my computer

**Reply by Zongmin741** [reply]
> add a line to setting.json { "notebook.output.wordWrap": true } it works on my computer

Thanks. It works for me too.
